conf/airframes/bebop_indi.xml

```diff
@@ -0,0 +1,268 @@
+<!DOCTYPE airframe SYSTEM "airframe.dtd">
+
+<airframe name="bebop">
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="bebop"/>
+
+    <target name="nps" board="pc">
+      <subsystem name="fdm" type="jsbsim"/>
+    </target>
+
+    <!--define name="USE_SONAR" value="TRUE"/-->
+
+	<!-- Subsystem section -->
+    <subsystem name="telemetry" type="transparent_udp"/>
+    <subsystem name="radio_control" type="datalink"/>
+    <subsystem name="motor_mixing"/>
+    <subsystem name="actuators" type="bebop"/>
+    <subsystem name="imu" type="bebop"/>
+    <subsystem name="gps" type="furuno"/>
+    <subsystem name="stabilization" type="indi"/>
+<!--     <subsystem name="ahrs"          type="float_mlkf"/> -->
+    <subsystem name="ahrs"          type="int_cmpl_quat">
+   <!-- <subsystem name="ahrs" type="int_cmpl_quat">
+      <configure name="USE_MAGNETOMETER" value="FALSE"/>
+      <define name="AHRS_USE_GPS_HEADING" value="FALSE"/>
+    </subsystem>-->
+<!--       <configure name="SECONDARY_AHRS" value="int_cmpl_quat"/> -->
+    </subsystem>
+    <subsystem name="ins" type="extended"/>
+  </firmware>
+
+  <modules main_freq="512">
+<!--     <load name="geo_mag.xml"/> -->
+<!--     <load name="air_data.xml"/> -->
+<!--     <load name="send_imu_mag_current.xml"/> -->
+<!--    <load name="logger_file.xml">
+      <define name="FILE_LOGGER_PATH" value="/data/ftp/internal_000"/>
+    </load>-->
+  </modules>
+
+  <commands>
+    <axis name="PITCH" failsafe_value="0"/>
+    <axis name="ROLL" failsafe_value="0"/>
+    <axis name="YAW" failsafe_value="0"/>
+    <axis name="THRUST" failsafe_value="6000"/>
+  </commands>
+
+  <servos driver="Default">
+    <servo name="TOP_RIGHT" no="0" min="3000" neutral="3000" max="12000"/>
+    <servo name="TOP_LEFT" no="1" min="3000" neutral="3000" max="12000"/>
+    <servo name="BOTTOM_LEFT" no="2" min="3000" neutral="3000" max="12000"/>
+    <servo name="BOTTOM_RIGHT" no="3" min="3000" neutral="3000" max="12000"/>
+  </servos>
+
+  <section name="MIXING" prefix="MOTOR_MIXING_">
+    <define name="TRIM_ROLL" value="0"/>
+    <define name="TRIM_PITCH" value="0"/>
+    <define name="TRIM_YAW" value="0"/>
+    <define name="NB_MOTOR" value="4"/>
+    <define name="SCALE" value="255"/>
+    <define name="MAX_SATURATION_OFFSET" value="3*MAX_PPRZ"/>
+
+    <!-- Time cross layout (X), with order NW (CW), NE (CCW), SE (CW), SW (CCW) -->
```

Comment:
We should use the "correct" motor mixing for these x-type quads. See https://github.com/paparazzi/paparazzi/blob/master/sw/airborne/subsystems/actuators/motor_mixing_types.h#L54
Even better, directly set this common type here...